joomla/joomla-cms
# [html5] System administrator template

#### Summary of Changes

This is a redo of https://github.com/joomla/joomla-cms/pull/9842 for **system admin** only.

This PR does a basic conversion of **system admin** template to HTML5.
Also made some minor corrections:
- Use scriptVersion for template js in all files
- Use the API in inline css/js (except error page)
- Use ie9 or lower html5.js in all files.
#### Testing Instructions
- Install patch
- Add the following code to force the system template in the frontend (before https://github.com/joomla/joomla-cms/blob/staging/libraries/cms/application/administrator.php#L185) 

``` php
 $this->template = new stdClass;
 $this->template->template = 'system';
 $this->template->params   = new Registry;
```
- Go to admin and check if the the following pages are working properly (they don't have style):
  - normal/component (test any existent page)
  - error pages (test one non existent URL)
- Remove the code added in 2. and check all is again ok  and with style.

Check also the code changes.
#### Observations

All pages should now start with this code:

``` html
<!DOCTYPE html>
<html lang="en-gb" dir="ltr">
<head>
[...]
    <meta charset="utf-8" />
[...]
```
